src/com/twu/biblioteca/BibliotecaApp.java

```diff
@@ -223,3 +223,4 @@ public void quit() {
         System.out.println("Thanks for using Biblioteca. See you next time.");
     }
 }
+
```

Comment:
I see you made a lot of use of switch and if...else statements. Fair enough. Some would say it's a code smell of conditional complexity. You're a cool coder though aren't you. How about using something like MenuItem enum that uses the command pattern?